pynamodb/connection/signals.py

```diff
@@ -0,0 +1,50 @@
+"""
```

Comment:
Because they are being used in connection/base.py and not with the higher level api calls, I thought it made sense to put it in there. That way when you are registering it, you know you are registering with a lower level connection call.

But if you feel like it makes more sense to move it to the main module, I can move it. I went back and forth for awhile, so could go either way.